test/unit/messaging/process-return-message.test.js

```diff
@@ -1,5 +1,6 @@
 jest.mock('ffc-messaging')
 const processReturnMessage = require('../../../app/messaging/process-return-message')
+const settlement = JSON.parse(JSON.stringify(require('../../mock-settlement')))
```

Comment:
move under `beforeEach`

when we have it here it is loaded in once, i.e. when the test file is ran

the problem arise because it is an object and we pass by reference rather than value - if we make a change to this object: `settlement.value = -2000`, `delete settlement.settled`, `settled.newKey = 'a'` these changes will persist throughout the test file and be used by each test that calls the object

e.g.

test 1: check that when settlement has the right schema (we dnt actualy have this func'al but imagine)
const result = Joi.validate(settlement, makeBelieveSchemaDefinitionWithAllKeys)
expect(result.error).toBeUndefined() // this passes, all keys present

test 2: check that when settlement does not have settled field we have this .error key
delete settlement.settled
const result = Joi.validate(settlement, makeBelieveSchemaDefinitionWithAllKeys)
expect(result.error).toBeDefined() // this passes, it through a hizzy fit

test 3: same as test 1, I've accidentally copy-pasta'd it, we know this will pass anyway
const result = Joi.validate(settlement, makeBelieveSchemaDefinitionWithAllKeys)
expect(result.error).toBeUndefined() // this FAILS, it's missing settled from the test before


moving importing `settlement` into `beforeEach` solves this